src/main/java/uk/gov/hmcts/divorce/solicitor/event/SolicitorStatementOfTruthPaySubmit.java

```diff
@@ -138,7 +139,8 @@ private void updateRespondentDigitalDetails(CaseData caseData) {
     public SubmittedCallbackResponse submitted(CaseDetails<CaseData, State> details,
                                                CaseDetails<CaseData, State> beforeDetails) {
         final CaseData caseData = details.getData();
-        final int feesPaid = caseData.getPayments().stream()
+        final List<ListValue<Payment>> payments = caseData.getPayments() == null ? emptyList() : caseData.getPayments();
```

Comment:
Is this change covered by a unit test?

Reply:
it was existing code which was failing while testing on docker. I think there is a test when payments is present let me add a test when there are no payments

Reply:
this is done now